Dockerfile

```diff
@@ -1,94 +1,87 @@
 # build ironlib wrapper binaries
-FROM golang:1.22-alpine AS stage0
+FROM golang:1.22-alpine AS helper-binaries
 
 WORKDIR /workspace
 
 # copy the go modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
+COPY go.mod go.sum ./
 
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
 RUN go mod download
 
-# copy ironlib go sources
+# copy rest of go sources
 COPY . .
 
 # build helper util
-ARG TARGETOS TARGETARCH
-RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH \
-    go build -o getbiosconfig examples/biosconfig/biosconfig.go && \
-    install -m 755 -D getbiosconfig /usr/sbin/
+ENV CGO_ENABLED=0
+RUN go build -o /usr/sbin/getbiosconfig examples/biosconfig/biosconfig.go
+RUN go build -o /usr/sbin/getinventory examples/inventory/inventory.go
 
-RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH \
-    go build -o getinventory examples/inventory/inventory.go && \
-    install -m 755 -D getinventory /usr/sbin/
-
-FROM almalinux:9-minimal as stage1
-ARG TARGETOS TARGETARCH
-
-# copy ironlib wrapper binaries
-COPY --from=stage0 /usr/sbin/getbiosconfig /usr/sbin/getbiosconfig
-COPY --from=stage0 /usr/sbin/getinventory /usr/sbin/getinventory
+FROM almalinux:9-minimal AS base
+FROM base AS deps
 
 # import and install tools
 RUN curl -sO https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
-RUN microdnf install -y --setopt=tsflags=nodocs crypto-policies-scripts && \
```

Comment:
does the `tsflags=nodocs` not help as much here?

Reply:
not that I could tell, though I was looking for -docs packages to be installed like in other distros.